modules/ROOT/pages/class-loader-library-config.adoc

```diff
@@ -9,69 +9,28 @@
-  <fileset dir="${server.config.dir}/mylib/Alexandria" includes="*.jar" />
-</library>
-----
-
-== Configure an application to use a Java library that is on an Open Liberty server
-
-In some cases, you can't bundle a Java library with your application, for example, when the application is already packaged and does not include the library.
-
-In the following example, an application that is called `Scholar` needs access to a Java library that is called `Alexandria`, which is located in the `${server.config.dir}/mylib/Alexandria` directory:
-
-[source,xml]
-----
-<application id="scholar" name="Scholar" type="ear" location="scholar.ear">
-  <classloader privateLibraryRef="Alexandria" />
-</application>
-
-<library id="Alexandria" apiTypeVisibility="+third-party">
-  <fileset dir="${server.config.dir}/mylib/Alexandria" includes="*.jar" />
-</library>
-----
+- <<Configure applications to use a shared library, Configure applications to use a shared library>>
+- <<Configure global libraries, Configure global libraries>>
+- <<Access third-party APIs from an application>>
+- <<Configure an application to use a Java library that is on an Open Liberty server>>
 
 == Configure applications to use a shared library
 
-
-In containerized environments, typically only one application runs on an Open Liberty server. However, some existing configurations might run multiple applications on a single server. In such cases, Java libraries can be shared across multiple applications. All the applications can use the same classes at run time or each application can use a separate copy of those classes that is loaded from the same location. Common libraries, which are used by multiple applications on the server, are specified by the `commonLibraryRef` element. Private libraries, which copy the library classes from the server for use by a single application, are specified by the `privateLibraryRef` element. Private library class paths are appended to the application class loader class path, while a common library has its own class loader that the application delegates to.
+In containerized environments, typically only one application runs on an Open Liberty server. However, some non-containerized configurations might run multiple applications on a single server. In such cases, Java libraries can be shared across multiple applications. All the applications can use the same classes at run time or each application can use a separate copy of those classes that is loaded from the same location. Common libraries, which are used by multiple applications on the server, are specified by the `commonLibraryRef` element. Private libraries, which copy the library classes from the server for use by a single application, are specified by the `privateLibraryRef` element. Private library class paths are appended to the application class loader class path, while a common library has its own class loader that the application delegates to.
```

Comment:
Acrolinx suggestion - non-containerized -> non containerized

Reply:
non is not a word